src/arx7.js

```diff
@@ -119,6 +121,26 @@ export class ARX7 {
           this.client.say(from, `I'm a bot! Contact [${admins}] for help`);
           resolve();
         }
+      }
+
+      const helpRegex = /^[.!](?:h(?:elp)?|man)(?:\s(.+))?$/i;
```

Comment:
Hm... sometimes people join a channel and type !help or !rules to get help for the channel in general. Getting this message might not be very helpful.